homeassistant/components/autarco/entity.py

```diff
@@ -0,0 +1,20 @@
+"""Base entity for the Autarco integration."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import AutarcoDataUpdateCoordinator
+
+
+class AutarcoEntity(CoordinatorEntity[AutarcoDataUpdateCoordinator]):
+    """Defines a base Autarco entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: AutarcoDataUpdateCoordinator,
+    ) -> None:
+        """Initialize the Autarco entity."""
+        super().__init__(coordinator)
```

Comment:
Can be removed. I think this PR could also be just be a base sensor entity. I am not quite sure why we noted that as a base class is not required per se